### PR TITLE
Fronter compatibility

### PR DESCRIPTION
**Pull request by DrDudelsack**
Changes program to read configuration settings from configuration.txt generated by fronter, rather then typing into the console

### CODE REVIEW

File: ImperatorToCk2/src/main/java/ImperatorToCK2/Processing.java [hunk 1]
```diff
@@ -1,707 +1,716 @@
...
+            flag = 1;
+
+        }
+
+        return flag;
+    }
+
+    public static String formatSaveName(String name) {// replaces /:*?"<>|- and space with _, while removing tabs and .rome
+        String quote = '"'+""; // " character, Java doesn't like isoated " characters
+        String tab = "    ";
```
Comment (IhateTrains) [inline ImperatorToCk2/src/main/java/ImperatorToCK2/Processing.java]: This is actually a string of 4 spaces. I doubt you intended this?

Comment (DrDudelsack) [reply ImperatorToCk2/src/main/java/ImperatorToCK2/Processing.java]: Indeed, lol, my IDE replaced hard tab characters with spaces. I'll try to upload a corrected version sometime tonight or tomorrow

File: ImperatorToCk2/src/main/java/ImperatorToCK2/Processing.java [hunk 2]
```diff
@@ -1,707 +1,736 @@
...
+                        if (provinceMappings[aqq].split(",")[0].equals("ck2")){
+                            impProv = provinceMappings[aqq].split(",")[1];
+                            aqq = 99999;
+
+                        }
+                        else {aqq = aqq + 1;}  
+
+                    }
+                    aqq = 0;   
+                    if (impProv != "a") {
```
Comment (IhateTrains) [inline ImperatorToCk2/src/main/java/ImperatorToCK2/Processing.java]: My Java knowledge is poor, but wouldn't

```suggestion
                    if (!"a".equals(impProv)) {
```

for string inequality checks be better?

Comment (DrDudelsack) [reply ImperatorToCk2/src/main/java/ImperatorToCK2/Processing.java]: I'm not sure, but `impProv != "a"` works and doesn't cause any errors. If it works, it works

**Pull request closed without merging**